Agents/HeatEmissionAgent/HeatEmissionAgent/src/main/java/uk/ac/cam/cares/jps/agent/heat/EndpointConfig.java

```diff
@@ -0,0 +1,41 @@
+package uk.ac.cam.cares.jps.agent.heat;
+
+import com.cmclinnovations.stack.clients.blazegraph.BlazegraphEndpointConfig;
+import com.cmclinnovations.stack.clients.docker.ContainerClient;
+import com.cmclinnovations.stack.clients.postgis.PostGISEndpointConfig;
+
+public class EndpointConfig {
+    private PostGISEndpointConfig postGISEndpointConfig;
+    private BlazegraphEndpointConfig blazegraphEndpointConfig;
+
+    public EndpointConfig() {
+        ContainerClient containerClient = new ContainerClient();
+        this.blazegraphEndpointConfig = containerClient.readEndpointConfig("blazegraph",
+                BlazegraphEndpointConfig.class);
+        this.postGISEndpointConfig = containerClient.readEndpointConfig("postgis", PostGISEndpointConfig.class);
+    }
+
+    public String getKgurl(String namespace) {
```

Comment:
Could you standardise the naming of the methods to getKgUrl
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Reply:
Done. Thanks for the recommendation.